scripts/checksum

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+var shell = require('shelljs');
+var helpers = require('./helpers');
+var crypto = require('crypto');
+
+function checksum(filename, assignment, rootChecksum) {
+  if(assignment) {
+    var assignmentConfig = shell.cat('exercises/' + assignment + '/' + filename).toString();
+    var assignmentChecksum = crypto.createHash('md5').update(assignmentConfig).digest("hex");
+    if(assignmentChecksum !== rootChecksum) {
+      shell.echo(filename + ' did not match for ' + assignment);
+      shell.exit(1);
+    }
+  }
+}
+
+function checkSumAll(filename, rootFileName = filename) {
+  shell.echo('\nChecking integrity of ' + filename + '...');
+
+  var config = shell.cat(rootFileName).toString();
+  var rootChecksum = crypto.createHash('md5').update(config).digest("hex");
```

Comment:
You might want to extract the two checksumers into their own function so it's easier to replace everywhere.